### ISSUE TITLE
Mac OSX issue with Requirement torch-1.2.0

### ISSUE THREAD

**Issue by Mikelew88** [fastaudio/fastai_audio#50]
Hi There,

I'm playing around with the tutorials locally on my mac for my own project but run into the following issue: 
```
RuntimeError: RuntimeError: fft: ATen not compiled with MKL support
...
The above operation failed in interpreter, with the following stack trace:
at .../site-packages/torchaudio/functional.py:203:14
        Fourier bins, and time is the number of window hops (n_frames).
    """
    assert waveform.dim() == 2

    if pad > 0:
        # TODO add "with torch.no_grad():" back when JIT supports it
        waveform = torch.nn.functional.pad(waveform, (pad, pad), "constant")

    # default values are consistent with librosa.core.spectrum._spectrogram
    spec_f = _stft(waveform, n_fft, hop_length, win_length, window,
             ~~~~~ <--- HERE
                   True, 'reflect', False, True)

    if normalized:
        spec_f /= window.pow(2).sum().sqrt()
    spec_f = spec_f.pow(power).sum(-1)  # get power of "complex" tensor
    return spec_f
```

This seems to be fixed in the latest torch release (https://github.com/pytorch/pytorch/issues/18996) but with the requirement of 0.1.2 I'm curious if anyone has a suggestion to get up and running on OSX. 

Thanks,
Mike

**Comment by mogwai** [comment]
We upgraded the torch version in a recent pull request have you tried to pull from master again recently?

We don't officially support mac. Best thing would be to spin up a virtual machine with Ubuntu on it or ssh into a hosted one. There are many tutorials on the fastai site.

I'll have a go to try and recreate the error although unless you have a cuda enabled graphics card your models will train very slowly.

**Comment by bigbeef105** [comment]
I have the same issue on Mac today..  I rolled back to 1.2/0.3 torchaudio to try to resolve a bug where FFT / spectrogram transform crashes the kernel every time.  Now this error.

**Comment by Mikelew88** [comment]
Yeah this was the latest on master, I just started working with the project yesterday. I'll go ahead and close this for now if Mac isn't really supported, and run on Ubuntu. It was the next step to get GPU acceleration anyways. Thanks for the responses.

**Comment by mogwai** [comment]
We created a colab notebook to get started with to if you don't have access to a ubuntu machine:

https://colab.research.google.com/drive/1s0Ouw5PxvrmHdm_gBU0qiA6piOf3VSWO

**Comment by mogwai** [comment]
Just realised the notebook isn't working. If somoene does get it working let me know and I'll add a link to the readme

**Comment by Mikelew88** [comment]
Looks like the issue is just that the cd command isn't working, not too
familiar with google collab notebooks, so not sure on the correct cd
syntax. You can just change the paths to be relative from home and it
works.

**Comment by mogwai** [comment]
Fixed this in #52
